ugvc/mrd/featuremap_consensus_utils.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import annotations
+
+from collections import defaultdict
+
+import numpy as np
+import pysam
+
+from ugvc.dna.format import AD, DP, GT, VAF
+from ugvc.mrd.featuremap_utils import FeatureMapFields
+
+fields_to_collect = {
+    "numeric_aggregated_fields": [FeatureMapFields.FILTERED_COUNT.value],
```

Comment:
I now realize that this is the same as X_FILTERED_COUNT, that has a different meaning in a regular FeatureMap, which is confusing. Could you change this to something like "ALT_COUNT"?

Reply:
I changed it to simply "rec_counter", in the end it's written into "AD[1]"